executor/aggregate.go

```diff
@@ -64,6 +68,8 @@ type HashAggPartialWorker struct {
 	// chk stores the input data from child,
 	// and is reused by childExec and partial worker.
 	chk *chunk.Chunk
+
+	trace opentracing.Span
```

Comment:
Could this kind of things be hidden in context.Context ?

Reply:
it will be used across different methods. Wrapping into ctx does not resolve this. So, I add this trace as part of the struct.

Reply:
What do you mean by saying "come with the channel message for a worker"? 

It will be really nice if you couple explains it more explicitly.